Makefile

```diff
@@ -466,7 +471,7 @@ e2e-image: ## Build the e2e manager image
 
 .PHONY: e2e
 e2e: e2e-image generate-e2e-templates
-e2e: $(GINKGO) $(KUSTOMIZE) $(KIND) $(GOVC) ## Run e2e tests
+e2e: $(GINKGO) $(KUSTOMIZE) $(KIND) $(GOVC) $(ENVSUBST) ## Run e2e tests
```

Comment:
Do we actually have to build envsubst here at this point?

Reply:
nope, removed